src/top.rs

```diff
@@ -332,14 +368,20 @@ impl Link {
                 let delay = self.delay(global_config.latency(), rand);
                 DeliveryStatus::DeliverAfter(self.now + delay)
             }
+            State::ExplicitPartitionAToB if src.ip() > dst.ip() => {
```

Comment:
This is a bit tricky to reason about as written. I assume this delivers in the non-partitioned link direction. Consider adding a comment on these branches.

Reply:
I agree. I'll see if I can make it clearer.

Reply:
I've tried to make it clearer.